test.js

```diff
@@ -93,7 +93,7 @@ describe('cpFile()', function () {
 
 	it('should not create dest on unreadable src', function (cb) {
 		cpFile('node_modules', 'tmp', function (err) {
-			assert(err.code === 'EISDIR');
+			assert(err.cause().code === 'EISDIR');
```

Comment:
What does `verror` provide us exactly? Couldn't we just modify the errors ourselves?